spec/asyncapi.md

```diff
@@ -538,7 +539,9 @@ Field Pattern | Type | Description
 {
   "user/signedup": {
     "subscribe": {
-      "$ref": "#/components/messages/userSignedUp"
+      "message": {
+        "$ref": "#/components/messages/userSignUp"
```

Comment:
Please note, that I also changed `userSignedUp` to `userSignUp` here and in other locations. Before, both variations where used inconsistently in the spec.

I decided for `userSignUp` because it was used more frequently and it's also the spelling used in the [Message Object](https://www.asyncapi.com/docs/specifications/v2.1.0#messageObject) section.

Reply:
Yeah, thanks for the heads up. IMHO it should be `userSignedUp` (past tense) but I don't think it matters a lot. Thanks for taking care anyway.

Reply:
+1 to rename it to use the simple past tense as it is common in events (Domain events, event-sourcing, cqrs, etc)

Reply:
I think originally, `userSignUp` was used, because the example message given in the [Message Object](https://www.asyncapi.com/docs/specifications/v2.1.0#messageObject) section is intended as a command. See for example this part: `"summary": "Action to sign a user up.",`.

In this case, past tense wouldn't be appropriate imho. So if we want to change to past tense, we should adapt a couple of other wordings in the spec. What do you think?

Reply:
We use both (command and event) interchangeably. I don't think we have to change anything here or in any other place. It's correct as it is, it was just a suggestion because most of the time we're speaking about events instead of commands. That's why we explicitly describe it when it's a command. For the rest of the occurrences, it should be an event. But really, not a blocker. It's ok as it is, at least for me.

Reply:
Alright, definitely not a hard opinion on the naming on my part. Even if it is not a blocker, since both of you have a preference towards the past tense names, I reverted the re-naming. So now, everything is as it was before naming wise.